spread/webconf/webconf-runs/task.yaml

```diff
@@ -0,0 +1,11 @@
+summary: Verify that webconf will start on an un-managed system by default
+execute: |
+    echo "The system should not be managed yet"
+    test `snap managed` = 'false'
+
+    echo "Verifying that webconf is running"
+    test `ps ax | grep -- -linux-gnu/webconf | grep -v grep | wc -l` -eq 1
```

Comment:
I'd have to test, but I'd imagine `ps h -C webconf` would work instead of `ps | grep | grep`